htdocs/contact/list.php

```diff
@@ -277,6 +282,54 @@
  * Actions
  */
 
+if ($action == "change") {	// Change customer for TakePOS
```

Comment:
Here a permission is missing, i suggest to add a test on 
$user->hasRight('takepos', 'run')

Reply:
hello @eldy; it is ok now; i've just added the permission test.